web/webkit/src/main/scala/net/liftweb/http/js/JsCommands.scala

```diff
@@ -646,7 +651,13 @@ trait HtmlFixer {
 }
 
 trait JsCmd extends HtmlFixer with ToJsCmd {
-  def &(other: JsCmd): JsCmd = JsCmds.CmdPair(this, other)
+  def &(other: JsCmd): JsCmd = {
+    if (other == JsCmds.Noop) {
+      this
+    } else {
+      JsCmds.CmdPair(this, other)
+    }
+  }
```

Comment:
It's still useful for cases where you do `Noop & ...`, e.g. when you use `Noop` for the start of a `fold` or whatever.

At least I think so ;)